### PR TITLE
Coding style

### PR DESCRIPTION
**Pull request by mlocati**

### CODE REVIEW

**Comment by itscaro** [comment]
@mlocati can you add .editorconfig as well?

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,14 @@ test-extensions-job-template: &test-extensions-job-template
 
 jobs:
   include:
+    - stage: Pre-test checks
+      name: Check coding style
+      if: type = pull_request
+      language: go
+      go: '1.13'
+      before_script: GO111MODULE=on go install mvdan.cc/sh/v3/cmd/shfmt
```
Comment (itscaro) [inline .travis.yml]:
```suggestion
      before_script: go install mvdan.cc/sh/v3/cmd/shfmt
```

Comment (itscaro) [inline .travis.yml]:
```suggestion
      before_script: go get github.com/mvdan/sh/cmd/shfmt
```

since go1.13 is used, i would remove GO111MODULE

Comment (mlocati) [reply .travis.yml]: :+1: see db53c3585ca6ee48f78893d3466b8378f3023624

Comment (mlocati) [reply .travis.yml]: :+1: see db53c3585ca6ee48f78893d3466b8378f3023624

**Comment by mlocati** [comment]
> @mlocati can you add .editorconfig as well?

:+1: see 001ac1dd607d8a4e647acafc42dca771dcb1b4db

File: .travis.yml (hunk 1 above)
Comment (mlocati) [reply .travis.yml]: It seems that `GO111MODULE` is required: see https://travis-ci.org/mlocati/docker-php-extension-installer/jobs/627782173#L227

**Pull request merged**